app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PublicRecipesController < ApplicationController
+  def index
+    @recipes = Recipe.where(public: true).order(created_at: :desc)
+  end
```

Comment:
- [ ] When I make a recipe public and load the page it become off again and no public recipes shown please make sure to fix that 
- 
<img width="257" alt="Screenshot 2022-12-16 at 11 02 56" src="https://user-images.githubusercontent.com/61790587/208075100-dfcd4a65-77aa-48a7-a13b-763861b0b580.png">

Reply:
## OPTIONAL 
* Well done 👏 working on the said issue but there is a bug here a new user should not be able to delete another user's recipe, I strongly suggest you fix this bug 🐛  [As stated in the requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=If%20the%20user%20is%20the%20owner%20of%20the%20recipe%2C%20should%20allow%20the%20user%20to%20delete%20it.)